src/main/java/com/flansmod/common/PlayerData.java

```diff
@@ -114,6 +121,38 @@ public void tick(EntityPlayer player)
 		if(shootClickDelay > 0)
 			shootClickDelay--;
 		
+		
+		// queued reloads
+		if(FlansMod.cancelReloadOnWeaponSwitch) {
+			if(gunToReload != null && !isHoldingGunToReload(player)) {
+				this.shootTimeLeft = 0;
+	        	this.shootTimeRight = 0;
+	        	
+	        	queuedReload = null;
+	        	gunToReload = null;
+				
+				if(player.worldObj.isRemote) {
+					FlansModClient.shootTimeRight = 0;
+		        	FlansModClient.shootTimeLeft = 0;
+		        	
+		        	GunAnimations gunAnimationRight = FlansModClient.getGunAnimations(player, false);
+		        	gunAnimationRight.reloadAnimationProgress = 0F;
+		        	gunAnimationRight.reloading = false;
+		        	GunAnimations gunAnimationLeft = FlansModClient.getGunAnimations(player, true);
+		        	gunAnimationLeft.reloadAnimationProgress = 0F;
+		        	gunAnimationLeft.reloading = false;
+				} 
+			} else if(!player.worldObj.isRemote && queuedReload != null) {
+				if(queuedReload.getReloadTime() > 0) {
```

Comment:
does the formatting look normal in your IDE? Idk if it's just Github that's made it look really weird

Reply:
Formatting looks off to me as well.